packages/path-proxy/index.test.js

```diff
@@ -6,6 +6,8 @@ const pathProxy = require("./index");
 
 describe("path-proxy", () => {
   beforeEach(() => {
+    process.env.webroot = "/var/www/spaship";
```

Comment:
The env var name is `SPASHIP_WEBROOT`.  Should this be `process.env["SPASHIP_WEBROOT"]` instead?

Reply:
It could be:)
both `SPASHIP_WEBROOT` and `WEBROOT` will works.
If there already has a `webroot` defined. It should use `SPASHIP_WEBROOT` to instead it